apis/mutations/v1alpha1/match.go

```diff
@@ -22,7 +22,7 @@ import (
 
 type Match struct {
 	Kinds              []Kinds                            `json:"kinds,omitempty"`
-	Scope              apiextensionsv1beta1.ResourceScope `json:"scope" protobuf:"bytes,4,opt,name=scope,casttype=ResourceScope"`
+	Scope              apiextensionsv1beta1.ResourceScope `json:"scope,omitempty"`
```

Comment:
If this is not provided, where do you set the default?

Reply:
To be pedantic: we don't set a default, it stays empty (kinda matters if you're thinking about UX of retrieving mutated resources, but probably not your concern here).

In our case, an unrecognized scope value is ignored by the matcher, which means all objects match it.

Reply:
That makes sense!